tracing-env-logger/Cargo.toml

```diff
@@ -13,7 +13,7 @@ tracing = "0.1"
 tracing-fmt = { path = "../tracing-fmt" }
 tracing-futures = { path = "../tracing-futures" }
 tracing-subscriber = { path = "../tracing-subscriber" }
-hyper = "=0.12.25"
```

Comment:
I don't remember exactly why this was ever an equals constraint (presumably the next version broke Rust 1.26 compat) but this needed to be relaxed as it was causing version mismatches when trying to use `tower-hyper` in the examples.